app/controllers/application_chatops_controller.rb

```diff
@@ -8,4 +8,13 @@ def verify_slack_hmac
   rescue Slack::Events::Request::MissingSigningSecret, Slack::Events::Request::InvalidSignature, Slack::Events::Request::TimestampExpired
     render plain: "Nope.", status: :unauthorized
   end
+
+  def subcommand
```

Comment:
This returns the subcommand for the given request. For `/doubleup list` this would be `"list"`